osgp/platform/osgp-core/src/main/java/org/opensmartgridplatform/core/application/services/EventNotificationMessageService.java

```diff
@@ -87,49 +90,6 @@ public void handleEvent(final String deviceIdentification, final EventNotificati
         this.handleEvent(deviceIdentification, dateTime, eventType, description, index);
     }
 
-    public void handleEvents(final String deviceIdentification, final List<EventNotificationDto> eventNotifications)
-            throws UnknownEntityException {
-
-        LOGGER.info("handleEvents() called for device: {} with eventNotifications.size(): {}", deviceIdentification,
-                eventNotifications.size());
-        for (final EventNotificationDto event : eventNotifications) {
-            LOGGER.info("  event: {}", event);
-        }
-
-        final Device device = this.eventNotificationHelperService.findDevice(deviceIdentification);
-
-        /*
-         * A list of bundled events may contain events that occurred over a
-         * period of time (and as such may contain multiple switching events per
-         * relay). Handling light switching events, only update the relay status
-         * once for the last switching in the list.
-         */
-        final List<Event> switchDeviceEvents = new ArrayList<>();
-        final List<Event> lightMeasurementDeviceEvents = new ArrayList<>();
-
-        for (final EventNotificationDto eventNotification : eventNotifications) {
-            final DateTime eventTime = eventNotification.getDateTime();
-            final EventType eventType = EventType.valueOf(eventNotification.getEventType().name());
-            final Event event = new Event(deviceIdentification,
-                    eventTime != null ? eventTime.toDate() : DateTime.now().toDate(), eventType,
-                    eventNotification.getDescription(), eventNotification.getIndex());
-
-            LOGGER.info("Saving event for device: {} with eventType: {} eventTime: {} description: {} index: {}",
-                    deviceIdentification, eventType.name(), eventTime, eventNotification.getDescription(),
-                    eventNotification.getIndex());
-            this.eventNotificationHelperService.saveEvent(event);
-
-            if (this.isSwitchingEvent(eventType)) {
-                switchDeviceEvents.add(event);
-            } else if (eventType.equals(EventType.LIGHT_SENSOR_REPORTS_DARK)
-                    || eventType.equals(EventType.LIGHT_SENSOR_REPORTS_LIGHT)) {
-                lightMeasurementDeviceEvents.add(event);
-            }
-        }
-
-        this.checkSwitchDeviceEvents(device, switchDeviceEvents);
-        this.checkLightMeasurementDeviceEvents(device, lightMeasurementDeviceEvents);
-    }
 
     private void checkSwitchDeviceEvents(final Device device, final List<Event> switchDeviceEvents)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "checkSwitchDeviceEvents" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)